cmd/gen-go-sample/main.go

```diff
@@ -271,6 +272,43 @@ func (g *generator) genSample(ifaceName string, methConf GAPICMethod, regTag str
 
 	// Some parts of request object are from arguments.
 	for _, attr := range valSet.Parameters.Attributes {
+		if attr.ReadFile {
+			var varName string
+			if attr.SampleArgumentName != "" {
+				varName = snakeToCamel(attr.SampleArgumentName) + fileContentSuffix
+			} else {
+				varName, err = fileVarName(attr.Parameter)
+			}
+			if err != nil {
```

Comment:
consider having this check inside the else, which is where the error is generated.